pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/EtcdMetadataStore.java

```diff
@@ -109,9 +109,9 @@ public EtcdMetadataStore(String metadataURL, MetadataStoreConfig conf, boolean e
         try {
             this.client = newEtcdClient(metadataURL, conf);
             this.kv = client.getKVClient();
-            this.client.getWatchClient().watch(ByteSequence.from("\0", StandardCharsets.UTF_8),
+            this.client.getWatchClient().watch(ByteSequence.from("/", StandardCharsets.UTF_8),
```

Comment:
This modification is beyond the scope of the PR description. You need to add a test to protect this change.

Reply:
> This modification is beyond the scope of the PR description.

This is not true, The value here is meaningless to the old method, but the new method needs to fill in the key as prefix. Not change it and you will get the red CI.

Reply:
Oh I see.